src/main/java/uk/gov/hmcts/reform/civil/service/citizenui/RespondentMediationService.java

```diff
@@ -18,10 +18,14 @@ public DefendantResponseShowTag setMediationRequired(CaseData caseData) {
             if (multiPartyScenario.equals(ONE_V_ONE)) {
                 return getDefendantResponseShowTagFor1v1(caseData);
             } else if (caseData.isMultiPartyClaimant(multiPartyScenario)) {
-                return DefendantResponseShowTag.CLAIMANT_MEDIATION_TWO_V_ONE;
+                if (caseData.hasDefendantAgreedToFreeMediation()) {
```

Comment:
if there isnt any else here. might be best to include it with the parent if condition. Same for the one below.

Reply:
ignore, looking at the else. it might not want anything to be returned.